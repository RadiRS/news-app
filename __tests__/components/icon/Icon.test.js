import 'react-native';
import 'jest-styled-components';
import React from 'react';
import renderer from 'react-test-renderer';
import IconComponent from '../../../src/components/common/icon/Icon';

test('renders Icon component correctly', () => {
  const tree = renderer
    .create(<IconComponent name="ios-add" size={0} color="#fff" style={{}} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
