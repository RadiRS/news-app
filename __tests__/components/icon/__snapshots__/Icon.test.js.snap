// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Icon component correctly 1`] = `
<Text
  allowFontScaling={false}
  style={
    Array [
      Object {
        "color": "#fff",
        "fontSize": 0,
      },
      Array [
        Object {},
      ],
      Object {
        "fontFamily": "Ionicons",
        "fontStyle": "normal",
        "fontWeight": "normal",
      },
      Object {},
    ]
  }
>
  ï„‚
</Text>
`;
